### DISCUSSION TITLE
Step modification lost after powering down the TX

### DISCUSSION THREAD

**Discussion by Christian-ALLEGRE** [egzumer/uv-k5-firmware-custom#263] [Q&A]
Hi egzumer, 
Using your firmware V0.20.1 (but it was the same with v0.19), when you change the STEP who is at 6.25kHz at startup, it stays to the value you set it until you power the TX down. But after power it up again, the STEP is 6.25kHz again... On the original firmware, the step setting was persistent after a TX reboot. Can you fix this in a future release ?
Thanks for this wonderful job :)

**Comment by gartnl** [comment]
When you change the stepsize in a memory channel you have to save that channel again. On powerup the actual memory channel defaults are loaded. This is no different than the stock firmware.
However, if you change the stepsize in VFO mode, it will be saved for that F-range.

**Reply by Christian-ALLEGRE** [reply]
OK, it works as you says when I save the channel after changing the STEP.
I'm sorry, I was not aware about this difference between VFO mode and Memory mode. 
I thought that STEP was a global parameter available for the whole TX, and not Channel or F-Range linked. 
Thanks for your help and your quick reply :)
